data/tribes/initialization/amazons/units.lua

```diff
@@ -1596,15 +1604,25 @@ wl.Descriptions():new_tribe {
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an Amazon production site: Ferry Yard
             purpose = pgettext("amazons_building", "Builds ferries."),
-            -- TRANSLATORS: Note helptext for an Amazon production site: Ferry Yard
-            note = pgettext("amazons_building", "Needs water nearby.")
+            note = {
+               -- TRANSLATORS: Note helptext for an Amazon production site: Ferry Yard, part 1
+               pgettext("amazons_building", "Needs water nearby."),
+               -- TRANSLATORS: Note helptext for an Amazon production site: Ferry Yard, part 2
+               pgettext("building", "Roads and trees along the shoreline block access to water."),
+            }
          }
       },
       {
          name = "amazons_shipyard",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an Amazon production site: Shipyard
-            purpose = pgettext("amazons_building", "Constructs ships that are used for overseas colonization and for trading between ports.")
+            purpose = pgettext("amazons_building", "Constructs ships that are used for overseas colonization and for trading between ports."),
+            note = {
+               -- TRANSLATORS: Note helptext for an Amazon production site: Shipyard, part 1
+               pgettext("building", "Needs water nearby."),
```

Comment:
A pond will not do it for a ship. 
Alternatives:
- Needs wide open water nearby.
- Needs the sea nearby.

Reply:
I guess the vote is for one of the alternatives.

Reply:
I'd prefer needs wide open water nearby.